drivers/adc/adc_shell.c

```diff
@@ -89,32 +48,48 @@ LOG_MODULE_REGISTER(adc_shell);
 #define CMD_HELP_GAIN	"Configure gain.\n"
 #define CMD_HELP_PRINT	"Print current configuration"
 
-#define DEVICES(n) DEVICE_DT_INST_GET(n),
-#define ADC_HDL_LIST_ENTRY(dev_)					\
-	{								\
-		.dev = dev_,						\
-		.channel_config = {					\
-			.gain = ADC_GAIN_1,				\
-			.reference = ADC_REF_INTERNAL,			\
-			.acquisition_time = ADC_ACQ_TIME_DEFAULT,	\
-			.channel_id = 0,				\
-		},							\
```

Comment:
What was wrong with this?

Reply:
I touched the macro define and let clang-format run over it to fix the indentation of the line breaks. I intentionally skipped eveything else clang-format changed in this file and staged only formatting changes where I changed something by myself.